tck/src/main/java/jakarta/json/bind/tck/ShrinkwrapHelper.java

```diff
@@ -1,57 +0,0 @@
-package jakarta.json.bind.tck;
```

Comment:
I confirmed with Andy that the intent was to remove this file as part of the pull.  The file is unused.